pages/models/regulations/governors/GovSteamEU/index.md

```diff
@@ -0,0 +1,422 @@
+for the proportional $$K_\mathrm{\omega\,cor}$$ [2] [4].
+
+#### Grid connected mode ($$f$$-part)
+
+In grid connected mode, the input is the measured electrical frequency
+$$f$$. The plant is linked to an upstream grid and contributes to overall
+grid stability. Here, frequency control is achieved through collective
+behavior; individual output adjustments are made via droop regulation.
+
+This is implemented via a proportional gain $$K_\mathrm{f\,cor}$$ with
+dead-band $$\Delta f_\mathrm{db}$$ contributing to the power set-point
+$$P_\mathrm{set}$$. The measured power $$P_\mathrm{e}$$ is then subtracted
+to obtain the error signal $$\Delta f$$ of the PID controller [2] [4].
+
+#### Emergency speed control and intercept valve (IV)
+
+**Emergency speed control**
+
+In case of loss of loads, the turbine speed instantly begins to
+increase. Ideally, the governor is able to handle such fluctuations and
+reduces the control valve set-point fast enough. If not, an emergency
+speed control is activated to reduce the active power output of the
+turbine fast.
+
+The *Emergency Speed Control* block in
+<a href="#fig-control_diagram" class="quarto-xref">Figure 1</a> outputs
+1 if $$\omega < \omega_\mathrm{max\,1}$$ and 0 if
+$$\omega > \omega_\mathrm{max\,2}$$. In-between it interpolates between 1
+and 0. This output is supposed to be below 1 or equal to 0 if $$\omega$$
+reaches dangerous levels (dangerous for mechanical stability of the
+turbine). It is then passed to the two different low value select blocks
+(min()).
+
+This has the effect of reducing the control valve and boiler set-points
+($$P_\mathrm{set\,ctrl}$$, $$P_\mathrm{set\,boiler}$$) fast, independently
+of what the controllers and the power set-point $$P_\mathrm{set}$$ do.
+
+This fast reduction of $$P_\mathrm{set\,ctrl}$$ also affects the intercept
+valve model, see below.
+
+**Intercept valve (IV)**
+
+Intercept valves remain fully open (set-point and position = 1) for
+regular speed/load adjustments [2]. In overspeed situations, the
+intercept valve shall close quickly and then reopen once normal
+conditions are restored.
+
+In this model, the intercept valve set-point is generated by the
+*intercept valves set-point generation* block. The input is the control
+valves input signal.
+
+If the control valve set-point $$P_\mathrm{set\,ctrl}$$ is above
+$$P_\mathrm{simx}$$, the output is 1. If $$P_\mathrm{set\,ctrl}$$ drops
+below, the output is reduced linearly until it reaches 0. In other
+implementations, a step to 0 is used instead of linear reduction as
+shown in the
+<a href="#fig-control_diagram" class="quarto-xref">Figure 1</a> from
+[1].
+
+In combination with the emergency speed control block, this leads to a
+fast reduction of the intercept valve set-point in case of over speed
+and hence interrupting the steam flow into the LP turbine and quickly
+reducing $$P_\mathrm{m}$$.
+
+## Model schema
+
+
+![](./drawings/GovSteamEU.drawio.svg)
+
```

Comment:
what does the green color is for?

Reply:
Just for style reasons, I can remove it.